pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -124,6 +126,8 @@ private TableConfigUtils() {
   // this is duplicate with KinesisConfig.STREAM_TYPE, while instead of use KinesisConfig.STREAM_TYPE directly, we
   // hardcode the value here to avoid pulling the entire pinot-kinesis module as dependency.
   private static final String KINESIS_STREAM_TYPE = "kinesis";
+
+  @Deprecated(since = "1.5.0", forRemoval = true)
```

Comment:
This isn't in the `spi` module so why can't we just directly remove it?

Reply:
I'm not sure if any client/third party code is using it, but sure I think I can remove it since the behavior is wrong